src/resources/help/en_US/relnotes3.3.1.html

```diff
@@ -34,6 +34,7 @@
   <li>Blueprint output for the x ray image query now supports file familying.</li>
   <li>In the color table window, the popup menu accessed via clicking on the default continuous color table button now correctly only displays continuous color tables, and the popup menu accessed from clicking on the default discrete color table button now correctly only displays discrete color tables.</li>
   <li>Added a color table searching feature to help users more easily find the color tables they wish to use.</li>
+  <li>Added support for MFEM Low Order Refinement into L2 Finite Element Spaces.</li>
```

Comment:
This reads a bit specific to someone converscent in MFEM verbiage. Is the functionality really only applicable to someone familiar with MFEM? If not, maybe find a way to word it more plainly. Sorry but I don't have any re-word to propose as I myself am not fully sure what it means.

Reply:
I updated it a bit; I also do not understand the MFEM verbiage as well as I'd like to.